docs/_sidebar.md

```diff
@@ -20,6 +20,7 @@
   - [Catch outgoing emails locally](how-to-catch-outgoing-emails-locally.md)
   - [Test translations locally](how-to-test-translations-locally.md)
   - [Understand the curriculum file structure](curriculum-file-structure.md)
+  - [How to work on tutorials that use CodeRoad](how-to-work-on-tutorials-that-use-coderoad.md)
```

Comment:
```suggestion
  - [Work on tutorials w/ CodeRoad](how-to-work-on-tutorials-that-use-coderoad.md)
```

Shorter titles work better on the side nav. You may want to add a descriptive title on the page itself. And while you are here the whole section called "Optional Guides" should be renamed to resources really.

Reply:
Should be resolved 👍